MainWindow.xaml.cs

```diff
@@ -504,7 +512,7 @@ private void SetCurrentDiscType()
             }
 
             // Get the current optical disc type
-            _currentMediaType = Validators.GetDiscType(selected?.Key);
+            _currentMediaType = null;// Validators.GetDiscType(selected?.Key);
```

Comment:
Why is this being commented out now? I don't see _currentMediaType being set here anymore... which makes the entire next "If we have an invalid current type" section always call

Reply:
I commented it to avoid slow downs but I didn't push the commit to restore it!